diff --git a/PyInstaller/depend/bindepend.py b/PyInstaller/depend/bindepend.py
index c0121c5..bcf76a5 100644
--- a/PyInstaller/depend/bindepend.py
+++ b/PyInstaller/depend/bindepend.py
@@ -892,6 +892,14 @@ def get_python_library_path():
         if python_libname:
             return python_libname
 
+        # Anaconda places the Python library in the lib directory, so we
+        # search this one as well.
+        prefix = os.path.join(compat.base_prefix, 'lib')
+        for name in PYDYLIB_NAMES:
+            full_path = os.path.join(prefix, name)
+            if os.path.exists(full_path):
+                return full_path
+
     elif compat.is_unix:
         for name in compat.PYDYLIB_NAMES:
             python_libname = findLibrary(name)
