diff --git a/bootloader/wscript b/bootloader/wscript
index ba0a545..76b834a 100644
--- a/bootloader/wscript
+++ b/bootloader/wscript
@@ -488,8 +488,8 @@ def configure(ctx):
         # made by detect_arch() call below), otherwise it will be
         # automatically set to the toolchain's default value.
         if not os.environ.get('MACOSX_DEPLOYMENT_TARGET'):
-            ctx.msg('MacOS deployment target', 'Setting to 10.13')
-            os.environ['MACOSX_DEPLOYMENT_TARGET'] = '10.13'
+            ctx.msg('MacOS deployment target', 'Setting to 10.7')
+            os.environ['MACOSX_DEPLOYMENT_TARGET'] = '10.7'
         else:
             ctx.msg('MacOS deployment target', 'Already set - %s' % (
                     os.environ.get('MACOSX_DEPLOYMENT_TARGET')))
